cms/static/js/spec/video/transcripts/editor_spec.js

```diff
@@ -132,6 +143,16 @@ function($, Backbone, _, Utils, Editor, MetadataView, MetadataModel, MetadataCol
                     type: MetadataModel.GENERIC_TYPE,
                     value: 'OEoXaMPEzfM'
                 },
+                videoIDEntry = {
+                    default_value: 'test default value',
+                    display_name: 'Video ID',
+                    explicitly_set: true,
+                    field_name: 'edx_video_id',
+                    help: 'Specifies the video ID.',
+                    options: [],
+                    type: MetadataModel.GENERIC_TYPE,
+                    value: 'advanced tab video id'
+                },
```

Comment:
I don't think we should do that because in the tests, they are meant to be 2 model fields with different values so that we can test their synchronization on switching between tabs. Also, `transcripts.collection` contains the models for basic tab while `metadataCollection`  comprises the models for the advanced tab.